app/services/hawkular_proxy_service.rb

```diff
@@ -78,7 +82,7 @@ def data(query)
   rescue StandardError => e
     {
       :parameters => params,
-      :error      => ActionView::Base.full_sanitizer.sanitize(e.to_s) + " " + _("(Please check your Hawkular server)")
+      :error      => ActionView::Base.full_sanitizer.sanitize(e.to_s) + " " + _("(Please check your #{@db_name} server)")
```

Comment:
This can't work, you need to `_("(Please check your %{type} server)") % {:type => @db_name}`

Reply:
👍 Thanks, I do not think i18n ... , copied your fix :-)

Reply:
:) The only thing to keep it in mind really is that the `_` function can only translate it if it finds an exact match in it's db. So any interpolation has to be done on the result of `_`, not before.